docs/notebooks/quickstart.md

```diff
@@ -0,0 +1,206 @@
+---
+jupytext:
```

Comment:
You probably don't want this table here.

Reply:
This table is what marks the file as a myst-nb document. It does not appear in the rendered docs.